models/turbine_models/tests/sdxl_test.py

```diff
@@ -10,16 +10,12 @@
 import shutil
```

Comment:
There has been a lot of pipelining and helper function's added. We should start working to integrate smaller unit tests for each case

Reply:
I see what you mean with this. Testing these gets pretty ugly without some context manager but these tests are definitely getting overloaded and need to be reduced to unit tests.